ic-stable-structures/src/structure/cache/btreemap.rs

```diff
@@ -33,6 +33,11 @@ where
             cache: SyncLruCache::new(max_cache_items),
         }
     }
+
+    /// Returns the inner collection
```

Comment:
even if obvious, could you please add a comment to state that accessing the inner collection will bypass the cache?

Reply:
Added